features/retry.feature

```diff
@@ -0,0 +1,282 @@
+Feature: Retry flaky tests
+
+  Using the `--retry` flag will retry failing tests for the specified number of times
+  Additionally using the `--retryTagFilter` flag will re-run only tests matching the tag expression
+
+  @spawn
+  Scenario: running Cucumber JS with --retryTagFilter but no positive --retry will fail
+    When I run cucumber-js with `--retryTagFilter @flaky`
```

Comment:
Can you add a test using a more complex set of tags `@flaky and @e2e` or something like that